Source/MLX/Documentation.docc/Organization/logical.md

```diff
@@ -53,6 +53,7 @@ if (a < b).all().item() {
 - ``equal(_:_:stream:)``
 - ``greater(_:_:stream:)``
 - ``greaterEqual(_:_:stream:)``
+- ``isClose(_:_:rtol:atol:equalNaN:stream:)``
```

Comment:
What is this function? We don't have it in Python

Reply:
```
  m.def(
      "isclose",
      &isclose,
      "a"_a,
      "b"_a,
      py::pos_only(),
      "rtol"_a = 1e-5,
      "atol"_a = 1e-8,
      py::kw_only(),
      "equal_nan"_a = false,
      "stream"_a = none,
      R"pbdoc(
        isclose(a: array, b: array, /, rtol: float = 1e-05, atol: float = 1e-08, *, equal_nan: bool = False, stream: Union[None, Stream, Device] = None) -> array
```

https://github.com/ml-explore/mlx/blob/main/python/src/ops.cpp#L1693

Reply:
Should you not have it? :-)